--- ./extensions/mojo_bounce_handler.pl.orig	Mon Jun 30 03:20:18 2003
+++ ./extensions/mojo_bounce_handler.pl	Sat Jul 19 10:26:09 2003
@@ -10,6 +10,9 @@
 #Change! the lib paths
 use lib qw(
 
+ /var/lib/mojo
+ /usr/share/mojo
+
  /home/account/www/cgi-bin/mojo
  /home/account/www/cgi-bin/mojo/MOJO
  /home/account/www/cgi-bin/mojo/MOJO/perllib
--- ./extensions/mojo_send.pl.org	Tue Apr  1 22:07:35 2003
+++ ./extensions/mojo_send.pl	Sun Apr 13 02:42:48 2003
@@ -10,7 +10,7 @@
 
 
 
-my $lib = '/home/path/to/cgi-bin/mojo';
+my $lib = '/usr/share/mojo';
 
 
 
--- ./MOJO/Config.pm.org	Tue Apr  1 22:07:37 2003
+++ ./MOJO/Config.pm	Sun Apr 13 02:39:53 2003
@@ -106,7 +106,7 @@
 
 =cut
 
-$FILES = '/home/youraccount/mojo_files';
+$FILES = '/var/lib/mojo';
 
 =pod
 
@@ -136,7 +136,7 @@
 
 =cut
 
-$MOJO_URL ='http://www.changetoyoursite.com/cgi-bin/mojo/mojo.cgi';
+$MOJO_URL ='http://localhost/mojo/mojo.cgi';
 
 =pod
 
